seeds/_sector/referral/slugs_optimism_rewards_seed.csv

```diff
@@ -0,0 +1,4 @@
+tx_hash,referrer_address,referee_address,reward_amount_raw
+0x0abd64a44d380991edd8c3a8032f268b1a7abf0417cce7c5b77136e508f7aa58,0x4093eed436bd6a5320316af7de30059a58f70c4c,0xeb5dfe747e4f43b4841a99879291b9139fe46e92,5000000000000000
+0x42d2d8b08cad7dd93c67e723735f40fd6600eff37021014304809d7397b49456,0x6860a976ad57c6c41d3697682cdace90c8db3c26,0x0ed1f91769d6add12f3aca1229f96eea198ac0ed,5000000000000000
+0xf0fe77d88ccbd1331e27e9600a8326b4d5f7c0d7625be871dd7247e5d5bbc792,0x0000000000000000000000000000000000000000,0xcb57c082893450408d432b5d0bc38def93b26040,15000000000000000
```

Comment:
appreciate that you have also added an example with a null referrer_address. :+1: